manage/manage.js

```diff
@@ -113,7 +113,13 @@ function initGlobalEvents() {
   setupLivePrefs();
   sorter.init();
 
-  prefs.subscribe(['manage.newUI'], () => switchUI());
+  prefs.subscribe([
+    'manage.newUI',
+    'manage.newUI.favicons',
+    'manage.newUI.faviconsGray',
+    'manage.newUI.sort',
```

Comment:
Do we need this? It seems that `sort.js` also subscribes to this option. Haven't dig into this though:
https://github.com/openstyles/stylus/blob/a4c08084bf0da629af48d3a186778576e6f9bad7/manage/sort.js#L192

Reply:
Good catch!